package.json

```diff
@@ -176,7 +175,7 @@
     "serve": "^11.1.0",
     "shelljs": "^0.8.3",
     "shx": "^0.3.2",
-    "storybook-chromatic": "^3.4.1",
+    "storybook-chromatic": "^3.5.5",
```

Comment:
This helped dedupe some versions of jest 24

Reply:
Might as well update to v5, what do you think? (the v4 upgrade was painless on polaris-styleguide, I expect it'll be the same for v5)

Reply:
done!